### PR TITLE
Engine 1.4

### PR DESCRIPTION
**Pull request by nastasi-oq**
cherrypick of "french" bug fix from master
better versioning management for packager.sh

### CODE REVIEW

**Comment by micheles** [comment]
The branch must be refreshed. Probably there is a conflict in the changelog.

**Comment by micheles** [comment]
LGTM

File: openquake/risklib/__init__.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 __all__ = ["VulnerabilityFunction", "DegenerateDistribution", "classical"]
 
 # the version is managed by packager.sh with a sed
-__version__ = '0.7.1'
```
Comment (daniviga) [inline openquake/risklib/__init__.py]: Be careful, you are pushing into master the version here

Comment (nastasi-oq) [inline openquake/risklib/__init__.py]: Automatic merge isn't possible; __init__.py and changelog must be fixed by hand

**Comment by daniviga** [comment]
LGTM

**Pull request merged**